TechnologyAssessmentApp/TechnologyAssessmentApp/GenericTaskFiles/ArrayDataProvider.swift

```diff
@@ -0,0 +1,77 @@
+//
+//  ArrayDataProvider.swift
+//  TechnologyAssessmentApp
+//
+//  Created by Kshitij Godara on 16/08/18.
+//  Copyright © 2018 Nagarro. All rights reserved.
+//
+
+//var items: [[T]] = []
```

Comment:
why is this commented ? if add a comment to explain why it's commented or delete it if not needed

Reply:
Comment Removed